hack/ensure-clusterctl.sh

```diff
@@ -45,7 +45,7 @@ EOF
 }
 
 install_clusterctl() {
-    if [[ "${OSTYPE}" == "linux"* || "${OSTYPE}" == "darwin22"* ]]; then
+    if [[ "${OSTYPE}" == "linux"* || "${OSTYPE}" == "darwin"* ]]; then
```

Comment:
oui. this change, is not rellated to this PR, can you please push it in another PR, as this is not a trivial changes.

Reply:
Done here https://github.com/outscale/cluster-api-provider-outscale/pull/336